frontend/src/app/main/ui/dashboard/team_form.cljs

```diff
@@ -25,14 +25,14 @@
 
 (defn- on-create-success
   [_form response]
-  (let [msg "Team created successfuly"]
+  (let [msg "Team created successfully"]
```

Comment:
Use visible?

Reply:
hmm probably, and we have missing translation strings here. I will take a note on this.